meson.build

```diff
@@ -0,0 +1,50 @@
+#
+# See LICENSE for more information about licensing
+#  Copyright 2023
+#
+# Author: Diego Arturo Avila Torres <dandida95@gmail.com>
+#         Luis G. Leon Vega <lleon95@gmail.com>
+#
+#
+
+project(
+  'cynq',
+  'cpp',
+  version : '0.1.0',
+  default_options : ['warning_level=3',
+                     'cpp_std=c++17'])
+
+add_global_arguments(
+  '-Wno-ignored-qualifiers',
+  language : 'cpp'
+)
+
+
+#############################
+## Dependencies definition ##
+#############################
+
+# Directory variables
+common_dependencies_dir = meson.source_root() + '/builddir/common-dependencies'
```

Comment:
Why do you need this?

Reply:
it was in the example from isms-kernel, I followed that.

Reply:
removed